app/scripts/lib/crypto/recovery-keys.js

```diff
@@ -0,0 +1,151 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * This file contains utilities to help create, bundle and unbundle encrypted
+ * recovery key data.
+ *
+ * For more encryption details, check out
+ + https://github.com/mozilla/fxa-auth-server/blob/master/docs/recovery_keys.md
+ */
+const Base32 = require('./base32');
+
+function importFxaCryptoDeriver() {
+  return import(/* webpackChunkName: "fxaCryptoDeriver" */ 'fxaCryptoDeriver');
+}
+
+function required(object, name) {
+  if (! object) {
+    throw new Error(`Missing ${name}`);
+  }
+}
+
+const RECOVERY_KEY_VERSION = '\x01';
```

Comment:
in mulling over this more:
* should this be `'1'` (which base32-decodes as `0x01` / `'\x01'`)?
* should this be exported or otherwise surfaced (e.g., `getRecoveryVersion()`) to aid in verification checks?